src/Dialogs/BulkRenamer/RenamerModifier.vala

```diff
@@ -247,7 +247,7 @@ public class Files.RenamerModifier : Object {
                         dt = file.modified;
                         break;
                     case RenameDateSource.NOW:
-                        dt = (uint64)get_monotonic_time ();
+                        dt = 0;
                         break;
                     default: // Created
                         dt = file.info.get_attribute_uint64 (GLib.FileAttribute.TIME_CREATED);
```

Comment:
Is it fine to consider as the case of `RenameDateSource.NOW` if dt is 0?

I think the case of `default` at this line also might return 0, according to the valadoc of [GLib.FileInfo.get_attribute_uint64()](https://valadoc.org/gio-2.0/GLib.FileInfo.get_attribute_uint64.html):

`If the attribute does not contain an unsigned 64-bit integer, or is invalid, 0 will be returned.`

If this happens, dt is set to 0 and `get_formated_datetime()` would return the current datetime.

Reply:
Thanks for pointing that out. We can add a test for the presence of the TIME_CREATED attribute, but what would be the appropriate date if the attribute is absent?  Maybe it would be better to not show the option at all if the attribute is absent,